plugins/main/bridge/bridge.go

```diff
@@ -188,6 +188,48 @@ func ensureBridgeAddr(br *netlink.Bridge, family int, ipn *net.IPNet, forceAddre
 	return nil
 }
 
+func ensureVlanInterfaceAddr(br *netlink.Bridge, vlan, family int, ipn *net.IPNet, forceAddress bool) error {
```

Comment:
This function seems to be a full copy of `ensureBridgeAddr()` -- can share the logic? Maybe just move `ensureVlanInterface` around line 547?